router/http/src/main/scala/io/buoyant/router/http/StaticIdentifier.scala

```diff
@@ -0,0 +1,20 @@
+package io.buoyant.router.http
+
+import com.twitter.finagle.buoyant.Dst
+import com.twitter.finagle.{Dtab, Path}
+import com.twitter.finagle.http.Request
+import com.twitter.util.Future
+import io.buoyant.router.RoutingFactory
+import io.buoyant.router.RoutingFactory.{IdentifiedRequest, RequestIdentification}
+
+class StaticIdentifier(
+  prefix: Path,
+  path: Path,
+  baseDtab: () => Dtab = () => Dtab.base
+) extends RoutingFactory.Identifier[Request] {
+
+  def apply(req: Request): Future[RequestIdentification[Request]] = {
+    val dst = Dst.Path(prefix ++ path, baseDtab(), Dtab.local)
```

Comment:
consider caching `prefix ++ path` on the class (or better yet, just take a path in the constructor) so we don't have to do additional per-request work